source/base/qprojector.cc

```diff
@@ -1459,10 +1375,9 @@ QProjector<dim>::DataSetDescriptor::face(
           offset += scale[i] * quadrature[i].size();
 
       if (dim == 2)
-        return {
-          offset +
-          (combined_orientation == numbers::default_geometric_orientation) *
-            quadrature[quadrature.size() == 1 ? 0 : face_no].size()};
+        return {offset +
+                combined_orientation *
+                  quadrature[quadrature.size() == 1 ? 0 : face_no].size()};
```

Comment:
This could technically be combined with the simplex/pyramid/wedge code above: I'll handle that in a future PR.